e2e/rbd.go

```diff
@@ -713,6 +713,106 @@ var _ = Describe("RBD", func() {
 				}
 			})
 
+			By("create a PVC and bind it to an app encrypted RBD volume and validate LUKS format", func() {
```

Comment:
Is this extra test case needed?

Reply:
This was part of my previous changes, where I also had one for LUKS1 :-)
Thought we can reuse some part of the work. If you feel it adds no value we can drop, but just give it a thought please.

Reply:
I do not think this needs extra validation, we do not care what the default LUKS format is?

Reply:
Sure, ok to remove.